getMongoData/getMongoData.js

```diff
@@ -302,6 +302,27 @@ function printDataInfo(isMongoS) {
                     }
                     printInfo('Indexes',
                               function(){return db.getSiblingDB(mydb.name).getCollection(col).getIndexes()}, section);
+                    printInfo('Index Stats',
+                              function(){var res = db.getSiblingDB(mydb.name).runCommand( {
+                                aggregate: col,
+                                pipeline: [
+                                  {$indexStats: {}},
+                                  {$group: {_id: "$key", stats: {$push: {accesses: "$accesses.ops", host: "$host", since: "$accesses.since"}}}},
+                                  {$project: {key: "$_id", stats: 1, _id: 0}}
+                                ]});
+
+                              if (res.hasOwnProperty('result')) {
+                                res.result.forEach(
+                                  function(d){
+                                    d.stats.map(
+                                      function(d){
+                                        return d.since = d.since.toUTCString();
```

Comment:
The `return` here isn't great, even though in this case it doesn't hurt.  Please remove it and change the above `d.stats.map` to `d.stats.forEach`.